modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -4695,7 +4713,8 @@ public CreateSnapshotCallable(
                     cacheGroupNames,
                     false,
                     onlyPrimary,
-                    dump
+                    dump,
+                    false
```

Comment:
This must be "compress" flag, this code is invoked when command starts from client node. Let's add test for that.